lioncore/metametamodel/lioncore.json

```diff
@@ -0,0 +1,2196 @@
+{
+  "serializationFormatVersion": "1",
+  "languages": [
+    {
+      "key": "LIonCore-M3",
+      "version": "1"
+    }
+  ],
+  "nodes": [
+    {
+      "id": "-id-LIonCore-M3",
+      "concept": {
+        "language": "LIonCore-M3",
+        "version": "1",
+        "key": "Language"
+      },
+      "properties": [
+        {
+          "property": {
+            "language": "LIonCore-M3",
```

Comment:
Did we not agree on using underscores instead of minusses? Or is that at some other place?

Reply:
I checked, and in previous versions we user underscores. I prefer underscores, because they are allowed in almost every environment within identifiers, while minusses are not.

Reply:
In #91 we decided to change _keys_ to use minuses (rationale in the issue).